atlas/instance_operations.py

```diff
@@ -305,6 +305,11 @@ def switch_settings_files(instance):
     else:
         tmp_path = '/tmp'
 
+    if 'google_tag_client_container_id' in instance['settings']:
+        google_tag_client_container_id = instance['settings']['google_tag_client_container_id']
+    else:
+        google_tag_client_container_id = 'tacocat'
```

Comment:
Guessing this is leftover from debugging, I think it should be something like
```suggestion
        google_tag_client_container_id = None
```